app/controllers/search_attempt_controller.rb

```diff
@@ -1,64 +1,66 @@
 class SearchAttemptController < ApplicationController
-    def create
-        owner = current_user.nil? ? false : current_user.owner
-        query = params[:search]
-        # Some of these objects may be nil, based on the search type
-        work = Work.find(params[:work_id]) if params[:work_id].present?
-        collection = Collection.find(params[:collection_id]) if params[:collection_id].present?
-        document_set = DocumentSet.find(params[:document_set_id]) if params[:document_set_id].present?
-        
-        if params[:work_id].present?
-            search_type = "work"
-        elsif (params[:collection_id].present? || params[:document_set_id].present?) && params[:search_by_title].present?
-            search_type = "collection-title"
-            query = params[:search_by_title]
-        elsif params[:collection_id].present? || params[:document_set_id].present?
-            search_type = "collection"
-        else # Find a Project search
-            search_type = "findaproject"
-        end
+  def create
+    owner = current_user.nil? ? false : current_user.owner
+    query = params[:search]
+    # Some of these objects may be nil, based on the search type
+    work = Work.find(params[:work_id]) if params[:work_id].present?
+    collection = Collection.find(params[:collection_id]) if params[:collection_id].present?
+    document_set = DocumentSet.find(params[:document_set_id]) if params[:document_set_id].present?
 
-        @search_attempt = SearchAttempt.new(
-            query: query,
-            search_type: search_type,
-            work_id: work&.id,
-            collection_id: collection&.id,
-            document_set_id: document_set&.id,
-            user_id: current_user&.id,
-            owner: owner
-        )
-        @search_attempt.save
-        session[:search_attempt_id] = @search_attempt.id
-        ajax_redirect_to(@search_attempt.results_link)
+    if params[:work_id].present?
+      search_type = 'work'
+    elsif (params[:collection_id].present? || params[:document_set_id].present?) && params[:search_by_title].present?
+      search_type = 'collection-title'
+      query = params[:search_by_title]
+    elsif params[:collection_id].present? || params[:document_set_id].present?
+      search_type = 'collection'
+    else # Find a Project search
+      search_type = 'findaproject'
     end
 
-    def show
-        @search_attempt = SearchAttempt.find_by(slug: params[:id])
-        
-        unless @search_attempt.nil?
-            if session[:search_attempt_id] != @search_attempt.id
-                session[:search_attempt_id] = @search_attempt.id
-            end
+    query = query&.strip
```

Comment:
This is mainly the fix. We could opt to trip ONLY leading spaces, but I would assume trailing spaces would present just the same problems so we trip both.